solutions/usaco-944.mdx

```diff
@@ -178,4 +174,46 @@ int main() {
 
 </CPPSection>
 
+<PySection>
+
+```python
+with open('fenceplan.in') as read:
+	n, m = map(int, read.readline().split())
+	connection = [[] for _ in range(n)]
+	coors = [[int(i) for i in read.readline().split()] for _ in range(n)]
+	for _ in range(m):
+		x, y = map(int, read.readline().split())
+		connection[x - 1].append(y - 1)
+		connection[y - 1].append(x - 1)
+
+visited = [0] * n
+ans = float('inf')
+
+for i in range(n):
+	if not visited[i]:
+		queue = [i]
+		visited[i] = 1
+
+		ax = bx = coors[i][0]
+		ay = by = coors[i][1]
+
+		for node in queue:
+			for conn in connection[node]:
+				if not visited[conn]:
+					visited[conn] = 1
+                    
+					cur_x, cur_y = coors[conn]
+
+					ax, ay = min(ax, cur_x), min(ay, cur_y)
+					bx, by = max(bx, cur_x), max(by, cur_y)
+
+  					queue.append(conn)
```

Comment:
This looks like an indentation error.

Reply:
I don't think there are any indentation errors though?
<img width="869" alt="Screen Shot 2022-05-23 at 1 41 38 AM" src="https://user-images.githubusercontent.com/52589420/169708352-6c5fa310-df62-4858-90cf-c023c2f7b150.png">

Reply:
It seems like the first two characters on line 210 are spaces, but everything else is indented with tabs. It looks off on my end since I use width-2 tabs.

Reply:
okay i'll re-tab